lockbox-ios/Action/RouteAction.swift

```diff
@@ -5,28 +5,25 @@
 import Foundation
 
 class RouteActionHandler : ActionHandler {
```

Comment:
Nit: whitespace

I think I'd suggest you getting `swiftlint` or `swiftfmt` into your build process.

Reply:
Followup issue: https://github.com/mozilla-lockbox/lockbox-ios/issues/107

Reply:
👍